### PR TITLE
IA Pages - show meta_id as id in the JSON endpoints and update frontend accordingly

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
@russellholt 
screenshot-maria duckduckgo com 5000 2015-04-21 18-15-44
screenshot-maria duckduckgo com 5000 2015-04-21 18-45-01
screenshot-maria duckduckgo com 5000 2015-04-21 18-45-24

### CODE REVIEW

File: lib/DDGC/DB/Result/InstantAnswer.pm [hunk 1]
```diff
@@ -15,14 +15,14 @@ sub u { [ 'InstantAnswer', 'view', $_[0]->id ] }
...
-    for_endpt => 1
 };
 primary_key 'id';
 
 # editable ID
 column meta_id => {
     data_type => 'text',
-    for_endpt => 1
+    for_endpt => 1,
+    alias => 'id'
```
Comment (russellholt) [inline lib/DDGC/DB/Result/InstantAnswer.pm]: alias?

Comment (MariagraziaAlastra) [inline lib/DDGC/DB/Result/InstantAnswer.pm]: yeah, new custom attribute :) it's used to display meta_id as id in the JSON endpoint which use the TO_JSON method to get the columns, such as ia_json

Comment (MariagraziaAlastra) [inline lib/DDGC/DB/Result/InstantAnswer.pm]: (ia_json --> single page JSON endpoint)

**Comment by russellholt** [comment]
:+1:  let's do it!

**Pull request merged**